server/src/main/kotlin/no/nav/pleiepengerbarn/uttak/server/LogFilter.kt

```diff
@@ -1,133 +1,37 @@
 package no.nav.pleiepengerbarn.uttak.server
 
 import jakarta.servlet.FilterChain
-import jakarta.servlet.ServletException
-import jakarta.servlet.http.Cookie
-import jakarta.servlet.http.HttpFilter
+import jakarta.servlet.ServletRequest
+import jakarta.servlet.ServletResponse
 import jakarta.servlet.http.HttpServletRequest
-import jakarta.servlet.http.HttpServletResponse
 import org.slf4j.LoggerFactory
 import org.slf4j.MDC
-import java.io.IOException
+import org.springframework.core.Ordered.LOWEST_PRECEDENCE
+import org.springframework.core.annotation.Order
+import org.springframework.stereotype.Component
+import org.springframework.web.filter.GenericFilterBean
 import java.util.*
-import java.util.function.Supplier
 
-// Kopieret klasse fra familie-felles å oppgradert til jakarta
-// https://github.com/navikt/familie-felles/blob/3cc4a074374717a7a916bdc683f91bd7abec984a/log/src/main/java/no/nav/familie/log/filter/LogFilter.kt
-class LogFilter(
-    /**
-     * Filter init param used to specify a [&lt;Boolean&gt;][Supplier]
-     * that will return whether stacktraces should be exposed or not
-     * Defaults to always false
-     */
-    private val exposeErrorDetails: Supplier<Boolean> = Supplier { false },
-    private val serverName: String? = null) : HttpFilter() {
+@Component
+@Order(LOWEST_PRECEDENCE)
+class LogFilter: GenericFilterBean() {
```

Comment:
Jeg er usikker på om denne blir tatt i bruk fordi du tok bort
```
@Bean
    fun logFilter(): FilterRegistrationBean<LogFilter> {
```
i app. Jeg tror den må være en FilterRegistrationBean for å funke, men ikke sikker